mystyle.css

```diff
@@ -0,0 +1,88 @@
+
+.container{
+		margin: 10px;
+		min-width: 320px;
+	    border:1px solid;
+	    list-style-type: none;
+	    overflow:hidden;
+	}
+	li{
+		float: left;
+		min-width: 300px;
+		width: 95%;
+		height: 400px;
+		border: 1px solid;
+		margin: 0 0px 47px -10px;
+		
+	}
+	
+	img{
+		width: 200px;
+		height: 200px;
+		padding: 0 0 0 25%;
+	}
+
+@media only screen and (min-width: 992px) {
+.container{
+		margin: 30px;
+	    border:1px solid;
+	    list-style-type: none;
+	    overflow:hidden;
+	}
+	li{
+		float: left;
+		width: 46%;
+		height: 400px;
+		border: 1px solid;
+		margin-bottom: 47px;
+		
+	}
+	li:nth-child(2n) {
+		margin-left: 2%;
+		margin-right: 20px;
+		
+	}
+	li:nth-child(2n-1) {
+		margin-right: 2%;
+		margin-left: 5px;
+		
+		
+	}
+	img{
+		width: 200px;
+		height: 200px;
+		padding: 0 0 0 25%;
+	}
+}
+@media only screen and (min-width: 1200px) {
+.container{
+		margin: 25px;
+	    border:1px solid;
+	    list-style-type: none;
+	    overflow:hidden;
+	}
+	li{
+		float: left;
+		width: 31%;
+		height: 400px;
+		border: 1px solid;
+		margin-bottom: 47px;
+		
+	}
+	li:nth-child(3n-1) {
+		
+		margin: 0 2% 0 2%;
+	}
+	li:nth-child(3n-2) {
+		margin: 0 0 0 0;
+	}
+	li:nth-child(3n) {
+		margin: 0 0 47px 0;
+	}
+	
+	img{
+		width: 200px;
+		height: 200px;
+		padding: 0 0 0 25%;
+	}
```

Comment:
Some styles are repeating in each of the media.  Need to keep the code DRY ( https://en.wikipedia.org/wiki/Don%27t_repeat_yourself )